### DISCUSSION TITLE
Mindblowing!

### DISCUSSION THREAD

**Discussion by ExclusiveRishi** [community/community#7022] [Copilot Conversations]
Just from 2 lines of comments, it generated almost accurate code, which could've taken me at least half an hour to write.

**Comment by CarloCattano** [comment]
Same, even on a personal badly written comment I had, fixed the thing right away in React

**Reply by Urethramancer** [reply]
It's getting scary lately. I was editing a file where the majority of the code was written, and I had a tiny, boring addition to make. There was barely any reference to those things, and lots of references to the functions I had written. I dreaded the task, as it would be boring, but after going to the bottom of the file and pressing enter it suggested exactly the damn lengthy  thing I was about to write.

So yeah, sometimes it impresses.

**Comment by jsbinette** [comment]
I'm also quite amazed.
I had tried the sister thing on openai but it was hard to use with measuring inputs and outputs.  It wasn't clear how to bring in files, etc.
I happened to be writing some exploratory stock market code this morning.  The thing was suggesting VERY pertinent things.  So much so that I think I saved 4 hours today!  Love it.
(it did make a curious mistake later today that was curious (Python). It tried to access a diction with () instead of [])
**I think the application to the adobe suite could also be just as important.  MS could make a billion a year from Adobe!**
